wai-handler-hal.cabal

```diff
@@ -55,8 +54,8 @@ common deps
     , hal                   >=0.4.7     && <0.4.11 || >=1.0.0 && <1.2
     , http-media            ^>=0.8.1.1
     , http-types            ^>=0.12.3
-    , network               >=2.8.0.0   && <3.2
-    , text                  ^>=1.2.3    || >=2.0   && <2.2
+    , network               ^>=3.2.0.0
```

Comment:
Is this package broken on old versions of `network`? If not, leave the wider lower bounds in place so that other users don't have to download the absolute latest versions of everything.

Reply:
Done.